src/renderer/transforms/forge.ts

```diff
@@ -1,7 +1,13 @@
+import * as path from 'path';
 import { execSync } from 'child_process';
 import { Files, PACKAGE_NAME } from '../../interfaces';
 import { getElectronBinaryPath } from '../binary';
 
+function getForgeVersion(): string {
+  const o = require(path.join(__dirname, '../../package.json'));
+  return o.devDependencies['@electron-forge/cli'];
```

Comment:
Nit: more descriptive variable name
```suggestion
  const fiddlePackageJSON = require(path.join(__dirname, '../../package.json'));
  return fiddlePackageJSON.devDependencies['@electron-forge/cli'];
```

Reply:
> (assuming you've tested that the require works)

Sample output:

```json
{
  "name": "witty-believe-thaw-y3yh4",
  "productName": "witty-believe-thaw-y3yh4",
  "description": "My Electron application description",
  "keywords": [],
  "main": "./main.js",
  "version": "1.0.0",
  "author": "charles",
  "scripts": {
    "start": "electron-forge start",
    "package": "electron-forge package",
    "make": "electron-forge make",
    "publish": "electron-forge publish",
    "lint": "echo \"No linting configured\""
  },
  "dependencies": {},
  "devDependencies": {
    "electron": "13.1.7",
    "@electron-forge/cli": "^6.0.0-beta.59",
    "@electron-forge/maker-deb": "^6.0.0-beta.59",
    "@electron-forge/maker-rpm": "^6.0.0-beta.59",
    "@electron-forge/maker-squirrel": "^6.0.0-beta.59",
    "@electron-forge/maker-zip": "^6.0.0-beta.59"
  },
  "config": {
    "forge": {
      "packagerConfig": {},
      "makers": [
        {
          "name": "@electron-forge/maker-squirrel"
        },
        {
          "name": "@electron-forge/maker-zip",
          "platforms": [
            "darwin"
          ]
        },
        {
          "name": "@electron-forge/maker-deb",
          "config": {}
        },
        {
          "name": "@electron-forge/maker-rpm",
          "config": {}
        }
      ]
    }
  }
}
```